templates/openstack/what-is-openstack.html

```diff
@@ -465,325 +586,301 @@ <h2 class="u-sv3">OpenStack is big</h2>
-                        attrs={"class": "p-heading-icon__img"},) |
-            safe }}
-            <h3 class="col-10 p-card__title p-heading--3">Charmed OpenStack</h3>
-          </div>
-        </div>
-        <hr class="p-rule--muted" />
-        <p class="p-card__content">
-          Charmed OpenStack is an enterprise cloud platform
-          engineered for price-performance that is designed to run mission-critical workloads.
-        </p>
-        <br />
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_1' -%}
+      <h3 class="p-heading--5">Charmed OpenStack</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_1' -%}
+      <div class="p-section--shallow">
         <p>
-          <a href="/openstack">Learn more about Charmed OpenStack&nbsp;&rsaquo;</a>
+          Charmed OpenStack is an enterprise cloud platform engineered for price-performance that is designed to run mission-critical workloads.
         </p>
-      </article>
-      <article class="col-6 p-card">
-        <div class="p-heading-icon">
-          <div class="p-heading-icon__header">
-            {{ image(url="https://assets.ubuntu.com/v1/85a9de76-ubuntu-icon.svg", alt="",
-                        width="50",
-                        height="50",
-                        hi_def=True,
-                        loading="lazy",
-                        attrs={"class": "p-heading-icon__img"},) |
-            safe }}
-            <h3 class="col-10 p-card__title p-heading--3">Ubuntu Server</h3>
-          </div>
+        <div class="p-cta-block">
+          <a href="/openstack">Learn more about Charmed OpenStack&nbsp;&rsaquo;</a>
         </div>
-        <hr class="p-rule--muted" />
-        <p class="p-card__content">
-          Ubuntu is the most popular Linux distribution across public clouds, data
-          centres, edge and IoT with a built-in security and compliance for
-          enterprise customers.
-        </p>
-        <br />
+      </div>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_2' -%}
+      <h3 class="p-heading--5">Ubuntu Server</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_2' -%}
+      <div class="p-section--shallow">
         <p>
+          Ubuntu is the most popular Linux distribution across public clouds, data centers, edge and IoT with a built-in security and compliance for enterprise customers.
+        </p>
+        <div class="p-cta-block">
           <a href="/server">Learn more about Ubuntu Server&nbsp;&rsaquo;</a>
+        </div>
+      </div>
+    {%- endif -%}
+  {%- endcall -%}
+
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="row--50-50">
+      <div class="col">
+        <h2>Public cloud costs increasing?
+      </div>
+      <div class="col">
+        <p>
+          Do not worry! Using multi-cloud architecture allows you to optimize your infrastructure costs. Our team of cloud experts will bootstrap and manage OpenStack private cloud for you.
```

Comment:
Left a [question](https://docs.google.com/document/d/15xrgIq_KUwHaCyFoI7-yS_JmBbsoyXJ4GpRMFAa7_is/edit?disco=AAABnzspEJk) on the doc, AFAIK we aren't supposed to use exclamation marks in our copy